DESCRIPTION

```diff
@@ -0,0 +1,26 @@
+Package: statusboard
+Title: PEcAn Status Board
+Version: 0.0.0.9000
+Authors@R: person('AUTHOR_FIRST', 'AUTHOR_LAST', email = 'AUTHOR@MAIL.COM', role = c('cre', 'aut'))
+Description: PKG_DESC.
+License: MIT + file LICENSE
```

Comment:
Author and description needs to be filled in. For license we've used NCSA instead of MIT